execution/src/posix/execution.cc

```diff
@@ -139,8 +144,37 @@ namespace leatherman { namespace execution {
         return fs.st_mode & S_IXOTH;
     }
 
-    string which(string const& file, vector<string> const& directories)
+    bool is_Builtin(string const& file)
```

Comment:
```suggestion
    bool is_builtin(string const& file)
```

snake case should be all lowercase